modules/custom/az_news/config/install/core.entity_view_display.node.az_news.az_feature.yml

```diff
@@ -0,0 +1,264 @@
+langcode: en
+status: true
+dependencies:
```

Comment:
```suggestion
dependencies:
  enforced:
    module:
      - az_news
```

Reply:
@cpoff what do you think about this suggestion?

Reply:
Oh, it looks you already applied manually.

Reply:
I need to spend some time looking at this a little more, but I'm tied up with Alumni right now.